lib/src/frame.dart

```diff
@@ -56,6 +80,42 @@ final _firefoxSafariFrame = RegExp(r'^'
     // empty in Safari if it's unknown.
     r'$');
 
+// With names:
+//
+// g@http://localhost:8080/test.wasm:wasm-function[796]:0x143b4
+// f@http://localhost:8080/test.wasm:wasm-function[795]:0x143a8
+// main@http://localhost:8080/test.wasm:wasm-function[792]:0x14390
+//
+// Without names:
+//
+// @http://localhost:8080/test.wasm:wasm-function[796]:0x143b4
+// @http://localhost:8080/test.wasm:wasm-function[795]:0x143a8
+// @http://localhost:8080/test.wasm:wasm-function[792]:0x14390
+//
+// Matches named groups:
+//
+// - "member": Function name, may be empty: `g`.
+// - "uri": `http://localhost:8080/test.wasm`.
+// - "index": `796`.
+// - "offset": (in hex) `143b4`.
+final _firefoxWasmFrame = RegExp(r'^(?<member>.*)@(?:(?<uri>.*):wasm-function'
+    r'\[(?<index>\d+)\]:0x(?<offset>[0-9 a-f A-F]+))$');
+
+// With names:
+//
+// <?>.wasm-function[g]@[wasm code]
```

Comment:
(Not immediately obvious that `<?>` is the literal text, and not a placeholder for something unknown. I had to read the test to figure it out. Might be worth emphasizing.)